app/src/main/res/drawable/fullscreen_exit.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
```

Comment:
Please increase the height and width of the `fullscreen_exit` drawable to `36dp`. Current icon can hardly be noticed.

Reply:
Missed it. Fixed. Thanks again. :)